code/modules/jobs/job_types/city/workshop.dm

```diff
@@ -24,8 +24,8 @@ Workshop employee
 /datum/job/chef/after_spawn(mob/living/carbon/human/H, mob/M, latejoin = FALSE)
 	H.set_attribute_limit(0)
 	ADD_TRAIT(H, TRAIT_WORK_FORBIDDEN, JOB_TRAIT)
-	to_chat(M, "<span class='userdanger'>Your workshop is northwest, of the city. </span>")
-	to_chat(M, "<span class='userdanger'>Make weapons for the fixers who come asking in exchange for money. You cannot give away weapons for free </span>")
+	job_important = "Make weapons for the fixers who come asking in exchange for money. You cannot give away weapons for free."
+	job_notice = "Your workshop is northwest, of the city."
```

Comment:
> Chef

Reply:
Nah, this job only exists on Ecorp

Reply:
> Nah, this job only exists on Ecorp

Yes *but* this is in the file for workshop job

Reply:
OH GOD